### PR TITLE
This PR fixes warning: already initialized constant warnings

### PR DESCRIPTION
**Pull request by rap1ds**

### CODE REVIEW

File: spec/services/paypal_service/store/paypal_payment_spec.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 describe PaypalService::Store::PaypalPayment do
   order = {
-    order_total: 100,
```
Comment (bladealslayer) [inline spec/services/paypal_service/store/paypal_payment_spec.rb]: Why was this changed?

Comment (rap1ds) [inline spec/services/paypal_service/store/paypal_payment_spec.rb]: This fixes a warning also. There's a `order_total` redefined in line 12, and that line overrides this definition.
